go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/RHsyseng/operator-utils v0.0.0-20191024171829-7e918ca09e5e
 	github.com/aerogear/unifiedpush-operator v0.0.0-20191212165307-3ea5e7aa705f
 	github.com/coreos/prometheus-operator v0.29.0
-	github.com/eclipse/che-operator v0.0.0-20190620145712-2f639261d8b5
+	github.com/eclipse/che-operator v0.0.0-20191122191946-81d08d3f0fde
```

Comment:
I can see that this is the commit that corresponds to the 2.0.0.GA branch, but I'm confused why it's still saying `v0.0.0` :thinking: 

@davidffrench thoughts? Is that because the che-operator is not a "module project" yet, or is it because of the `.GA` suffix on the tag, or something else?

Reply:
@grdryn I am under the impression this happens when the dependency version does not specifically have the format `v`+`x.x.x`. Some dependencies don't include the `v` in their git tags.

Reply:
Seems likely. from [here](https://github.com/golang/go/wiki/Modules#modules):

> Modules must be semantically versioned according to semver, usually in the form v(major).(minor).(patch), such as v0.1.0, v1.2.3, or v1.5.0-rc.1. The leading v is required. If using Git, tag released commits with their versions.